Testscripts/Linux/build_test_vpp.sh

```diff
@@ -78,21 +78,15 @@ function build_test_vpp () {
 	ssh "${1}" "cd ${VPP_DIR} && sed -i '/vpp_uses_dpdk_mlx5_pmd/s/^# //g' build-data/platforms/vpp.mk"
 	ssh "${1}" "cd ${VPP_DIR} && sed -i '/vpp_uses_dpdk_mlx4_pmd/s/^# //g' build-data/platforms/vpp.mk"
 
-	# VPP 18.10 or higher is supported for Azure
-	# VPP 19.04 or higher supports 18.11 or 19.02 dpdkVersion
-	# VPP 19.01 supports 18.08 or 18.11 dpdkVersion
-	# VPP 18.10 supports 18.05 or 18.08 dpdkVersion
-	ssh "${1}" "cd ${VPP_DIR} && make pkg-${package_type} DPDK_VERSION=${dpdkVersion} vpp_uses_dpdk_mlx4_pmd=yes vpp_uses_dpdk_mlx5_pmd=yes DPDK_MLX4_PMD=y DPDK_MLX5_PMD=y DPDK_MLX5_PMD_DLOPEN_DEPS=y"
-	check_exit_status "make -j pkg-{package_type} DPDK_MLX5_PMD=y DPDK_MLX4_PMD=y DPDK_MLX5_PMD_DLOPEN_DEPS=y on ${1}" "exit"
+	ssh "${1}" "cd ${VPP_DIR} && make pkg-${package_type} vpp_uses_dpdk_mlx4_pmd=yes vpp_uses_dpdk_mlx5_pmd=yes DPDK_MLX4_PMD=y DPDK_MLX5_PMD=y DPDK_MLX5_PMD_DLOPEN_DEPS=y DPDK_MLX4_PMD_DLOPEN_DEPS=y"
+	check_exit_status "make pkg-${package_type} vpp_uses_dpdk_mlx4_pmd=yes vpp_uses_dpdk_mlx5_pmd=yes DPDK_MLX4_PMD=y DPDK_MLX5_PMD=y DPDK_MLX5_PMD_DLOPEN_DEPS=y DPDK_MLX4_PMD_DLOPEN_DEPS=y on ${1}" "exit"
```

Comment:
Removed -j?

Reply:
it is for log purpose, I make the log and actual command consistent.